homeassistant/components/huawei_lte/select.py

```diff
@@ -0,0 +1,127 @@
+"""Support for Huawei LTE selects."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass, field
+from functools import partial
+import logging
+
+from huawei_lte_api.enums.net import LTEBandEnum, NetworkBandEnum, NetworkModeEnum
+
+from homeassistant.components.select import (
+    DOMAIN as SELECT_DOMAIN,
+    SelectEntity,
+    SelectEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import UNDEFINED
+
+from . import HuaweiLteBaseEntityWithDevice
+from .const import DOMAIN, KEY_NET_NET_MODE
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class HuaweiSelectEntityDescription(SelectEntityDescription):
+    """Class describing Huawei LTE select entities."""
+
+    setter_fn: Callable[[str], None] = lambda x: None
```

Comment:
Should this rather avoid a default value to make sure all future PRs will not forget defining it? This would require converting this to a mixin and inheriting the `HuaweiSelectEntityDescription` from it and from the `SelectEntityDescription`.